package/display/package

```diff
@@ -58,10 +60,12 @@ display() {
             ln -s "$libname" "$pkgdir"/opt/lib/"${libname%.*.*}"
             ln -s "$libname" "$pkgdir"/opt/lib/"${libname%.*}"
             install -D -m 644 -t "$pkgdir"/lib/systemd/system "$srcdir"/rm2fb.service
+            install -D -m 644 -t "$pkgdir"/lib/udev/rules.d/ "$srcdir"/15-systemd-input.rules
```

Comment:
This will only include the udev rule in the rM2 display package. Shouldn’t we include it in the rM1 package too?